### ISSUE TITLE
Feature Request - Ability to Sort Custom Fields Alphabetically When Viewing

### ISSUE THREAD

**Issue by davesmith87** [amidaware/tacticalrmm#1840]
**Describe the solution you'd like**
When right clicking on a client, and choosing "edit", you can view Custom Fields for Client.  This custom fields appear to be added incrementally when they were created.  Add a button to "sort" these custom fields alphabetically.

**Comment by ZzBombardierzZ** [comment]
Bump.

Please, @wh1te909 🙏

**Comment by P6g9YHK6** [comment]
i think the sorting issues need to be "sorted" at every level... site, client, custom fields, agent, url action, checks, tasks, fav scripts we should be able to add "priority value" to each and have the ui sort them by smallest to biggest it's kind of a global issue

**Comment by ZzBombardierzZ** [comment]
@P6g9YHK6 definitely. I know the description mentions client but I am hopeful it would be at least all custom field views if not more
